deepmd/utils/summary.py

```
@@ -74,6 +74,31 @@ def __call__(self) -> None:
+                gpus = tf.config.list_physical_devices("GPU")
+                if gpus:
+                    # Use the first physical GPU device identifier as the device name
+                    device_name = gpus[0].name
+            elif backend == "Paddle":
+                import paddle
+
+                # Use Paddle's current device string (e.g., "gpu:0") as a device identifier
+                device_name = paddle.get_device()
+        except Exception:
+            # Best-effort device name detection; ignore failures silently
+            pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)